packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx

```diff
@@ -57,15 +57,17 @@ describe('<ElementAvatar>', () => {
       <ElementAvatar
         userId="@user-alice"
         displayName="Alice"
-        avatarUrl={'mxc://alice'}
+        avatarUrl={'mxc://example.com/imageAlice'}
       />,
     );
 
     await expect(
       screen.findByRole('presentation', { hidden: true }),
     ).resolves.toHaveAttribute(
       'src',
-      expect.stringMatching(/\/_matrix\/media\/r0\/thumbnail\/alice/i),
+      expect.stringMatching(
+        /https:\/\/.*\/_matrix\/media\/r0\/thumbnail\/example\.com\/.*imageAlice/i,
```

Comment:
the URI is still incorrect, even if we keep to the r0 version. please check with the spec: https://spec.matrix.org/latest/client-server-api/#get_matrixmediav3thumbnailservernamemediaid

Reply:
Also `.*` is a regexp wild card (match any char). Please remove it.